src/device/mod.rs

```diff
@@ -191,6 +191,9 @@ pub struct RequestedState {
 
     // Requested keepalive periods
     pub(crate) keepalive_periods: FeaturePersistentKeepalive,
+
+    // Indicates whether we want to use IPv6 addresses
+    pub use_ipv6: bool,
```

Comment:
I think this is duplicated, as you should have the same value in `self.features.ipv6`

Reply:
Yes, but actually no. I.e. I need to pass this flag somehow to wg-controller - it seems to me that we don't pass features in any objects and adding additional argument is not an option because clippy won't let us additional arguments. Of course, we could consider wrapping wg function arguments with some structs, but I'm not sure if that's worth it.